### PR TITLE
Create zarangollo.md

### PR DESCRIPTION
**Pull request by enrique95**
Zarangollo recipe put in /src

### CODE REVIEW

**Comment by enrique95** [comment]
Zarangollo recipe (1st submission)

File: src/zarangollo.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- Onion
+- Eggplant
+- Olive oil
+- Egg (optional)
```
Comment (hallzy) [inline src/zarangollo.md:14]: Having explicit amounts here would be useful. I assume from the context in the directions that this calls for 1 onion and 1 eggplant etc, but having explicit values would be ideal I think.

File: src/zarangollo.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Directions
+
+1. Cut onion in julianne (a third or half of an onion ring)
```
Comment (hallzy) [inline src/zarangollo.md:18]:
```suggestion
1. Cut onion in julienne (a third or half of an onion ring)
```

I assume you mean `julienne`? Or am I just wrong here.

File: src/zarangollo.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+2. Cut the eggplant in slices or chunks that can fit in a teaspoon
+3. Add oil to a pan (not a shallow one, wide and a bit deep is ideal if you have one) until it barely covers the vegetables
+4. Cook and stirr in medium heat until they are tender, caramelized and translucid
```
Comment (hallzy) [inline src/zarangollo.md:21]:
```suggestion
4. Cook and stir in medium heat until they are tender, caramelized and translucent
```

**Review by hallzy — CHANGES_REQUESTED** [review]

**Review by enrique95 — COMMENTED** [review]
Thank you for your suggestions and broken english repairs

**Comment by enrique95** [comment]
Thank you @hallzy for the suggestions!

**Review by hallzy — APPROVED** [review]

**Pull request merged**